ios/Capacitor/Capacitor/JS.swift

```diff
@@ -19,13 +19,6 @@ internal struct JSCall {
     let pluginId: String
     let method: String
     let callbackId: String
-
-    init(options: [String: Any], pluginId: String, method: String, callbackId: String) {
```

Comment:
What's this in here for?

Reply:
Because I run `npm run fmt`.
For `Swift` the struct's `init` function is default created when compiled, no need to create explicitly.

Reply:
Yeah - I'm not sure why they're in there - `npm run fmt` catching it probably makes this ok to include.